packages/plugins/graphql/services/schema/builders/index.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { merge, map, pipe, reduce } = require('lodash/fp');
+
+const enums = require('./enums');
+const dynamicZone = require('./dynamic-zones');
+
+const entity = require('./entity');
+const entityMeta = require('./entity-meta');
+const type = require('./type');
+
+const response = require('./response');
+const responseCollection = require('./response-collection');
+
+const queries = require('./queries');
+const mutations = require('./mutations');
+
+const filters = require('./filters');
+const inputs = require('./input');
+
+const buildersFactories = [
+  enums,
+  dynamicZone,
+  entity,
+  entityMeta,
+  type,
+  response,
+  responseCollection,
+  queries,
+  mutations,
+  filters,
+  inputs,
+];
+
+/**
+ * Instantiate every builder with the given context
+ * @param {object} context
+ * @param {object} context.strapi
+ * @parma {object} context.registry
+ */
+module.exports = context => {
+  return pipe(
+    // Create a new instance of every builders
+    map(factory => factory(context)),
+    // Merge every builder into the same object
+    reduce(merge, {})
```

Comment:
The reduce iteratee is actually invoked with two arguments since we're using `lodash/fp` now :slightly_smiling_face: 

See https://github.com/lodash/lodash/wiki/FP-Guide (reduce) for more information
![image](https://user-images.githubusercontent.com/25851739/124604716-80e09180-de6b-11eb-976d-e919beeaaded.png)